org.testeditor.web.backend.persistence/src/main/java/org/testeditor/web/backend/persistence/TestExecutorResource.xtend

```diff
@@ -0,0 +1,33 @@
+package org.testeditor.web.backend.persistence
+
+import javax.inject.Inject
+import javax.ws.rs.Consumes
+import javax.ws.rs.POST
+import javax.ws.rs.Path
+import javax.ws.rs.QueryParam
+import javax.ws.rs.core.MediaType
+import javax.ws.rs.core.Response
+import javax.ws.rs.core.UriBuilder
+import org.slf4j.LoggerFactory
+import javax.ws.rs.Produces
+
+@Path("/tests")
+@Consumes(MediaType.APPLICATION_JSON)
+class TestExecutorResource {
+
+	static val logger = LoggerFactory.getLogger(TestExecutorResource)
+
+	@Inject TestExecutorProvider executorProvider
+
+	@POST
```

Comment:
Why is it a post and not a get?

Reply:
Since a testexecution is created. Idea is that a test execution itself is a resource that can be created (may be even deleted, etc.)